priv/riak_kv.schema

```diff
@@ -90,14 +90,14 @@
 
 %% @doc Frequency to prompt exchange per vnode
 %% The number of milliseconds which the vnode must wait between self-pokes to
-%% maybe prompt the next exchange. Default is 4 minutes - check all partitions
-%% when n=3 every 30 minutes.
+%% maybe prompt the next exchange. Default is 8 minutes - check all partitions
+%% when n=3 once every hour (in each direction).
```

Comment:
probably add the formula on how to get from 8 and 3 to once every hour. That is not straigthforward

Reply:
Added - it is 1 + (n - 1) * n ticks per cycle.  there are n - 1 exchanges for each partition, and n partitions ... plus 1 tick to prompt the refill of the cycle